Flow.Launcher.Core/Resource/Theme.cs

```diff
@@ -320,35 +343,39 @@ private enum WindowCompositionAttribute
         /// </summary>
         public void SetBlurForWindow()
         {
+            if (BlurEnabled)
+            {
+                SetWindowAccent(Application.Current.MainWindow, AccentState.ACCENT_ENABLE_BLURBEHIND);
+            }
+            else
+            {
+                SetWindowAccent(Application.Current.MainWindow, AccentState.ACCENT_DISABLED);
+            }
```

Comment:
Maybe only change this if blur state is changed?

Reply:
in the original code this state is always set though via on_loaded everytime when flow is started, not sure if we really need to manage this

Reply:
@taooceros did you want to show me what you mean?

Reply:
Oh sorry I forget to reply. I mean only call this method when we switch from Blur to non-blur or non-blur to Blur. Also maybe remove the call of this method in MainWindow.OnLoaded and move that to ChangeTheme

Reply:
> call this method when we switch from Blur to non-blur or non-blur to Blur

there's not really a good way to tell what state blur was previously. is this just so the call is not done multiple times when user cycles through themes? i tested current way there hasnt been any issues.

>  remove the call of this method in MainWindow.OnLoaded and move that to ChangeTheme

this is already removed.